tx.go

```diff
@@ -348,7 +355,16 @@ func (l *LocalMapper) Begin(c *influxql.Call, startingTime int64, chunkSize int)
 			l.limit = math.MaxUint64
 		}
 	} else {
-		lit, _ := c.Args[0].(*influxql.VarRef)
+		// Check for calls like `derivative(mean(value), 1d)`
+		var nested *influxql.Call = c
+		if fn, ok := c.Args[0].(*influxql.Call); ok {
+			nested = fn
+		}
+
+		lit, ok := nested.Args[0].(*influxql.VarRef)
```

Comment:
Yes.  Without it and in the case of `select derivative(mean(value),..)`, it fails because `mean` is not a `*VarRef`. It needs to descend into the nested the call.